lambda_functions/alembic/versions/6b4e36091047_answer_options_added.py

```diff
@@ -0,0 +1,189 @@
+"""Answer Options added
```

Comment:
Sidenote: Should we define a standard naming convention for migrations?

Reply:
Yes